src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_commands.py

```diff
@@ -286,3 +292,12 @@ def load_flexibleserver_command_table(self, _):
                             client_factory=cf_postgres_flexible_private_link_resources) as g:
         g.command('list', 'list_by_server')
         g.custom_show_command('show', 'flexible_server_private_link_resource_get', custom_command_type=flexible_servers_custom_postgres)
+
+    with self.command_group('postgres flexible-server index-tuning', postgres_flexible_config_sdk,
+                            custom_command_type=flexible_servers_custom_postgres,
+                            client_factory=cf_postgres_flexible_config, table_transformer=table_transform_output_parameters) as g:
+        g.custom_command('update', 'index_tuning_update', custom_command_type=flexible_servers_custom_postgres)
+
+    with self.command_group('postgres flexible-server tuning-options', postgres_flexible_tuning_options_sdk,
+                            client_factory=cf_postgres_flexible_tuning_options) as g:
+        g.custom_command('show', 'tuning_options_get', custom_command_type=flexible_servers_custom_postgres)
```

Comment:
Same here. Can be `az postgres flexible-server show-tuning-options`

Reply:
More commands to come for tuning-options